docker-run-opencv-ovms.sh

```diff
@@ -116,24 +116,25 @@ else
 fi
 
 #todo: this will need to be updated to how we are supporting these configs
-if [ "${OCR_DISABLED}" == "0" ] && [ "${BARCODE_DISABLED}" == "0" ] && [ "${CLASSIFICATION_DISABLED}" == "0" ] && [ "${REALSENSE_ENABLED}" == "0" ]; then
-	pipeline="yolov5s_full.sh"
+# TODO: the followings should put into another pipeline script that can be configured in configuraiton.yaml so that it can be run in more unified way
+# if [ "${OCR_DISABLED}" == "0" ] && [ "${BARCODE_DISABLED}" == "0" ] && [ "${CLASSIFICATION_DISABLED}" == "0" ] && [ "${REALSENSE_ENABLED}" == "0" ]; then
+# 	pipeline="yolov5s_full.sh"
 	
-elif [ "${OCR_DISABLED}" == "1" ] && [ "${BARCODE_DISABLED}" == "1" ] && [ "${CLASSIFICATION_DISABLED}" == "1" ]; then
-	pipeline="yolov5s.sh"
-elif [ "${OCR_DISABLED}" == "1" ] && [ "${BARCODE_DISABLED}" == "1" ] && [ "${CLASSIFICATION_DISABLED}" == "0" ]; then
-	pipeline="yolov5s_effnetb0.sh"
-elif [ "${REALSENSE_ENABLED}" == "1" ]; then
-	# TODO: this will not work for diff pipelines like _full and _effnetb0 etc
-	pipeline="yolov5s_realsense.sh"
+# elif [ "${OCR_DISABLED}" == "1" ] && [ "${BARCODE_DISABLED}" == "1" ] && [ "${CLASSIFICATION_DISABLED}" == "1" ]; then
+# 	pipeline="yolov5s.sh"
+# elif [ "${OCR_DISABLED}" == "1" ] && [ "${BARCODE_DISABLED}" == "1" ] && [ "${CLASSIFICATION_DISABLED}" == "0" ]; then
+# 	pipeline="yolov5s_effnetb0.sh"
+# elif [ "${REALSENSE_ENABLED}" == "1" ]; then
+# 	# TODO: this will not work for diff pipelines like _full and _effnetb0 etc
+# 	pipeline="yolov5s_realsense.sh"
 	
-else
-	echo "Not implemented"
-	exit 2 
-fi
+# else
+# 	echo "Not implemented"
+# 	exit 2
+# fi
```

Comment:
We can remove commented out code for now and use the history to pull this if statement into a new script.

Reply:
yea, i'll remove it

Reply:
removed and pushed